README.md

```diff
@@ -64,6 +64,18 @@ With the following processes running as cron jobs on daily basis:
 
 On heroku these are configured using the scheduler.
 
+Monitoring
+----------
+This server will print log messages it is recommend run with `DEBUG` as
+`"* -superagent -6to5 -mocha:* -express:*"`.
+Notice that messages printed containing the substring `[alert-operator]` should
```

Comment:
I like having `alert-operator`, but it's really critical that we don't use it as the basis for how we monitor and track the systems.  I know it's hard to do anything better without writing a real logging system, but I'm just a little nervous about establishing a convention of a string appearing in logs as being something that's a critical error.  At the least, could we avoid typos by exporting this string from taskcluster-base?  I know that sounds silly, but it'd suck if someone printed the string `'[alert-opreator]'` and we missed a critical issue because of it.